components/core/DownloadDaemon.py

```diff
@@ -240,7 +239,7 @@ def on_error(ws, error):
 
 
 def on_close(ws):
-    pass
+    print("socket closed")
```

Comment:
Do we have a logger? Probably replace `print` with logging

Reply:
done